packages/crew_profile_base.rb

```diff
@@ -46,6 +45,10 @@ def self.postinstall
       FileUtils.rm "#{HOME}/.bashrc.bak" if File.exist? "#{HOME}/.bashrc.bak"
       FileUtils.rm "#{HOME}/.zshrc.bak" if File.exist? "#{HOME}/.zshrc.bak"
 
+      # dbus file moved to dbus postinstall, so remove it.
+      FileUtils.rm_f './src/env.d/env.d/03-dbus'
+      FileUtils.rm_f "#{CREW_PREFIX}/etc/env.d/03-dbus"
+
```

Comment:
Do we need this since the environment has been moved into install instead of postinstall?

Reply:
This will clean up the older installed dbus file to avoid running it twice.

Reply:
This file wasn't installed by dbus initially? :face_with_head_bandage:

Reply:
This is cleaning up the env.d/03-dbus file from crew_profile_base